assets/js/modules/analytics/components/setup/SetupForm.js

```diff
@@ -82,7 +83,9 @@ export default function SetupForm( { finishSetup } ) {
 	return (
 		<form className="googlesitekit-analytics-setup__form" onSubmit={ submitForm }>
 			<StoreErrorNotice moduleSlug="analytics" storeName={ STORE_NAME } />
+
 			<ExistingTagNotice />
+			<ExistingGTMPropertyNotice />
```

Comment:
See above, I think we should wrap this in a clause `if ( ! existingTag )`.

Reply:
Updated.